### PR TITLE
Added AddShoppingSmartAd example.

### PR DESCRIPTION
**Pull request by nwbirnie**

### CODE REVIEW

**Comment by nwbirnie** [comment]
Hey Ian, can you sign the CLA please (see link above)?

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/AddShoppingSmartAd.java [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+import com.google.protobuf.BoolValue;
+import com.google.protobuf.DoubleValue;
+import com.google.protobuf.Int64Value;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Collections;
+
+/**
+ * This example creates a Smart Shopping campaign, a Smart Shopping ad group and a Smart Shopping ad
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/AddShoppingSmartAd.java]: nit: too many "ands". Change "and a Smart Shopping ad group ad" to ", a Smart Shopping ad group ad"

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/AddShoppingSmartAd.java [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+/**
+ * This example creates a Smart Shopping campaign, a Smart Shopping ad group, a Smart Shopping ad
+ * group ad and a listing group for "All products".
+ *
+ * <p>Prerequisite: You need to have access to a Merchant Center account. You can find instructions
+ * to create a Merchant Center account here: https://support.google.com/merchants/answer/188924.
+ * This account must be linked to your AdWords account. The integration instructions can be found
+ * at: https://developers.google.com/adwords/shopping/full-automation/articles/t15.
+ */
+public class AddShoppingSmartAd {
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/AddShoppingSmartAd.java]: PTAL at the presubmit failure as well. Looks like poms may be incorrect. Error is:

    Non-resolvable parent POM for com.google.api-ads:google-ads-examples:0.4.1-SNAPSHOT:
    Could not find artifact com.google.api-ads:google-ads-parent:pom:0.4.1-SNAPSHOT
    and 'parent.relativePath' points at wrong local POM @ line 22, column 11

Comment (nwbirnie) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/AddShoppingSmartAd.java]: All sorted, thanks! Local branch was a few commits behind master.

**Pull request merged**